planet/cli/data.py

```diff
@@ -37,9 +37,24 @@ def data(ctx, base_url):
 
 def parse_item_types(ctx, param, value: str) -> List[str]:
     """Turn a string of comma-separated names into a list of names."""
-    # Note: we could also normalize case and validate the names against
-    # our schema here.
-    return [part.strip() for part in value.split(",")]
+    # convert user-input strings to match our naming schema
+    update_value = value.lower()
+
+    dict = {
```

Comment:
Two things here:
1. we want the CLI to be as thin as possible, that means, any validation and adjusting of case is something we want to do in the API, if possible. 
2. We want to avoid hard-coding values that derive from the API schema and instead pull them from the schema itself. See how this is done in the `specs.py` module, which is used by `order_requests.py`

For this effort, I would suggest creating a ticket for validation and normalization of item_types in the API and I would go without this logic in this PR.

Reply:
I am not seeing the resolution of these issues. Could you address these issues by either removing the validation code and creating/including a reference to the ticket or adding an explanation on how these issues have been resolved in another way?